.github/workflows/pull-request.yml

```
@@ -43,7 +43,7 @@ jobs:
       - name: Get changed TypeScript files
         id: changed-files
         uses: tj-actions/changed-files@v40
-      
+        
```

yamllint

[error] trailing spaces

(trailing-spaces)